src/index.html

```diff
@@ -1,12 +1,226 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.css">
-  </head>
-  <body>
-    <h1>Catalog</h1>
-  </body>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Catalog</title>
+  <link rel="stylesheet" href="./styles/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header">
+    <a href="#" class="header__logo">
+      <img src="images/logo.png" alt="logo MOYO">
+    </a>
+    <nav>
+      <ul class="nav">
+        <li><a href="#" class="nav__link is-active">apple</a></li>
```

Comment:
```suggestion
        <li><a href="#" class="nav__link nav__link--is-active">apple</a></li>
```
`is-active` - модифікатор

Reply:
виправив